{
  Throwable t=exceptionEvent.getCause();
  if (t != null && IOException.class.isAssignableFrom(t.getClass())) {
    LOG.error(t,t);
    LOG.debug(t,t);
  }
 else {
    LOG.error(t,t);
    LOG.debug(t,t);
  }
  ctx.sendUpstream(exceptionEvent);
}
